nemo/collections/tts/modules/common.py

```diff
@@ -255,7 +216,7 @@ def getRadTTSEncoder(
     encoder_n_convolutions=3,
     encoder_embedding_dim=512,
     encoder_kernel_size=5,
-    norm_fn=nn.BatchNorm1d,
+    norm_fn=MaskedInstanceNorm1d,
```

Comment:
Line 215: we should follow the naming convention for functions over class names. `getRadTTSEncoder` -> `get_radtts_encoder`.